server/views/probationPractitionerReferrals/interventionProgress.njk

```diff
@@ -54,9 +54,9 @@
     <p>Below you will be able to find the end of service report created by the service provider. Once submitted, you will be able to read and download it.</p>
   {% endif %}
 
-  {% if presenter.referralEndRequested %}
-    <p class="govuk-body">{{ presenter.referralEndRequestedText }}</p>
-  {% else %}
+  {% if presenter.canCancelReferral %}
     <a href='{{ presenter.referralCancellationHref }}'>Cancel this referral</a>
+  {% elseif presenter.referralEndRequested %}
```

Comment:
do we not want something here say if the referral was cancelled? and also something to tell them what this text actually is? like "service user died" on it's own might be a bit jarring?

Reply:
`referralEndRequestedText`, which we're currently showing is "You requested to end this service on [this date]". We don't actually state the reason. I think it's more of catch-all phrase as the SP might have to complete an EOSR in some cases. Do you think I should run this past Helen, or do you think it's safe for now?